packages/forma-36-website/gatsby-config.js

```diff
@@ -30,7 +33,7 @@ module.exports = {
       },
       {
         name: 'Guidelines',
-        link: '/guidelines/',
+        link: '',
```

Comment:
was this intended?

Reply:
Yup, but good reminder that I need to document this. Empty string links now act as section headers.